python/requirements.txt

```diff
@@ -1,6 +1,2 @@
-wheel==0.29.0
-enum34==1.1.2
-protobuf==2.6.1
-configparser
-netaddr==0.7.10
-bitarray==0.9.3
```

Comment:
Why you took out all the requirements from here?

Reply:
Some of them were needed because we were using python2 before (e.g., enum34), some are installed by default in the distros we'll currently be supporting (Ubuntu, CentOS). Having a second look now I'm not so sure about `bitarray` anymore. It's used in `starter.py` and I think I didn't use that script until now. I'll try it out and come back to this afterwards.

Reply:
@davvore33 I tried it out, added some additional fixes on top of the older commits. Now the starter.py script runs. However, I think the script itself might not be completely correct. I'm not sure though if anyone uses it. If they do maybe it's good to follow up and make sure everything works fine.

CI is running now but it looks like everything will pass.